src/StackAdmin/Azs.Fabric.Admin/readme.md

```diff
@@ -51,44 +51,128 @@ require:
   - $(repo)/specification/azsadmin/resource-manager/fabric/readme.md
 
 input-file:
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ApplicationOperationResults.json
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/ComputeOperationResults.json
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/Fabric.json
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/EdgeGateway.json
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/EdgeGatewayPool.json
-  - $(repo)/specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2016-05-01/FabricLocation.json
+  # (TODO) Add Compute and Network swagger files
```

Comment:
need to remove this comment

Reply:
I temporarily removed some spec files, because they are Network/Compute related and we can add them back when the fabric infra part is ready.